app/currency/views.py

```diff
@@ -34,9 +36,11 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class RateListView(ListView):
+class RateListView(FilterView):  # ListView was used for create list view,now it used filters view
     template_name = 'rate_list.html'
-    queryset = Rate.objects.all().select_related('bank')
+    queryset = Rate.objects.all().select_related('bank').order_by('id')
```

Comment:
`order_by('id') -> order_by('-id')`

Reply:
Rates ordering has changed to DESC. Resolved in https://github.com/Story-TellerX/currency/pull/19/commits/f960c5c15419232566e73e2d34de8318131b1d25